### DISCUSSION TITLE
2-Factor in Saved Accounts on VRCX

### DISCUSSION THREAD

**Discussion by tamponking** [vrcx-team/VRCX#738] [Q&A]
Hey, I have my accounts saved in VRCX I'm trying to log in into the website and it asks me to put my 2-factor authenticator code, I cannot get it cause my phone got completely smushed and needs a screen repair, Every time my IP changes I get logged out of VRChat unlike on VRCX which saves my 2-Factor code, I was asking if there's any temporarily way that I could do to log back into my account by looking towards the saved account details on VRCX, Would be very helpful and save me a lot of time to fix a broken phone screen..

Thanks for the honest support so far I admire using VRCX.

**Comment by Natsumi-sama** [comment]
It's a very hands on technical process that I don't have the time to explain it in detail, if you're technically inclined you'll be able to follow along.
Open `%AppData%\VRCX\VRCX.sqlite3` with https://sqlitebrowser.org/
Copy value contents in cookies table.
Decode as base64.
Manually add cookies to browser.

**Reply by tamponking** [reply]
That did not work whatsoever, I tried decoding and it gave me weird symbols with a few readable variables, Can you tell me the process of how to add the cookies into the browser??

Also a little side note: I have more than a single account so I don't know how would that work.

**Reply by tamponking** [reply]
![image](https://github.com/vrcx-team/VRCX/assets/110065928/a5cc3156-e22a-4c96-89fe-fd6f17661d48)

Nothing to replace this with whatsoever.
